docs/reference/offline-stores/README.md

```diff
@@ -1,16 +1,86 @@
 # Offline stores
 
-Please see [Offline Store](../../getting-started/architecture-and-components/offline-store.md) for an explanation of offline stores.
+Please see [Offline Store](../../getting-started/architecture-and-components/offline-store.md) for a conceptual explanation of offline stores.
```

Comment:
Since other README's don't have detailed explanations, wondering if we should have a separate page for most of this and keep this as mainly organization

Reply:
that's fair; I'll add an Overview page as we do [here](https://docs.feast.dev/getting-started/concepts/overview)